tracer/src/Datadog.Trace/Debugger/LiveDebugger.cs

```diff
@@ -196,6 +196,7 @@ internal void UpdateAddedProbeInstrumentations(IReadOnlyList<ProbeDefinition> ad
                                     lineProbes.Add(new NativeLineProbeDefinition(location.ProbeDefinition.Id, location.MVID, location.MethodToken, (int)location.BytecodeOffset, location.LineNumber, location.ProbeDefinition.Where.SourceFile));
                                     break;
                                 case LiveProbeResolveStatus.Unbound:
+                                    Log.Information("ProbeID {ProbeID} is unbound.", probe.Id);
```

Comment:
It already has written a few lines above where we log the probe id and the status (which is unbound in this case), do we need it here?

Reply:
This log message leads to easier testing